src/onkar/Assignment19.java

```diff
@@ -0,0 +1,55 @@
+/*1. Print the frequecy of given character from given String.
+input : technocredits
+        t
+		
+output : frequency of t in technocredits is 2.
+9
+2. Print character of given String in reverse order.
+input : techno
+output : 
+       o
+	   n
+       h
+       c
+	   e
+	   t*/
+
+package onkar;
+
+public class Assignment19 {
+	
+	void findFrequency(String input) {
+		int count = 0;
+		
+		for (int index=0; index<input.length(); index++) {
+			if (input.charAt(index)=='t')
```

Comment:
use generic process, your code will only work for character 't', you have to pass any character as a parameter and your code will print frequency of it